.github/workflows/build.yml

```
@@ -4,7 +4,7 @@ jobs:
   build:
     strategy:
       matrix:
-        java: [17, 19]
+        java: [ 17, 19 ]
```

seems like a bit of a pointless change to me  
I've never liked spacing arrays like that
```suggestion
        java: [17, 19]
```